README.rst

```diff
@@ -57,21 +57,20 @@ https://icepyx.readthedocs.io/en/latest/getting_started/install.html
 Examples (Jupyter Notebooks)
 ----------------------------
 
-Listed below are example jupyter-notebooks
+Listed below are example Jupyter notebooks
 
-`ICESat-2_DAAC_DataAccess_Example <https://github.com/icesat2py/icepyx/blob/main/examples/ICESat-2_DAAC_DataAccess_Example.ipynb>`_
+`IS2_data_access <https://icepyx.readthedocs.io/en/latest/example_notebooks/IS2_data_access.html>`_
```

Comment:
Debating on whether to have this as a relative link or as an absolute link. This will work fine now, but it will likely break in future icepyx versions if the filename or folder structure changes.

Reply:
Great point. Maybe we can open an issue for transitioning to relative links generally but at least it points to a rendered version (versus the "maybe-it-will-display" GH version) for now?

Reply:
Ok, just leave it like this for now then. I feel like Jupyter notebooks render a bit more reliably this year (perhaps because of https://github.blog/changelog/2022-01-07-jupyter-notebooks-and-specialized-file-formats-display-faster), but the rendered page on readthedocs is probably nicer to look at.

Reply:
Cool - I haven't been tracking the GitHub Jupyter-Notebook updates.